app/views/staffs/new.js.erb

```diff
@@ -0,0 +1,5 @@
+let staffNewForm = "<%= escape_javascript(render partial: 'new_staff') %>"
+let displayElementStaffNew = document.getElementById('user-forms')
+
```

Comment:
you need to add `closeModalListener('cancel_modal');` on line 3 otherwise the cancel button will create an assignment